backend/app/schemas/content.py

```
@@ -0,0 +1,38 @@
+import uuid
+from datetime import datetime
+from typing import Optional, List # List is not used in these specific schemas but good to have for future
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)